packages/api/tests/endpoints/test-granules.js

```diff
@@ -863,8 +863,14 @@ test.serial('DELETE deletes a granule that exists in PostgreSQL but not Elastics
     .expect(200);
 
   t.is(response.status, 200);
-  const { detail } = response.body;
-  t.is(detail, 'Record deleted');
+  const responseBody = response.body;
```

Comment:
These tests are 👍🏻  but we should also include a test/test update on `deleteGranuleAndFiles` in the lib tests, as it's a lib method and also used in bulk operations.